taipy/gui/_renderers/__init__.py

```diff
@@ -100,9 +100,9 @@ def __parse_file_content(self, content):
 
     def __sanitize_content(self, content: str) -> str:
         # replace all CRLF (\r\n) with LF (\n)
```

Comment:
I don't really see what was changed by this PR...

And to be honest, I would have written all this:
```
        # Replace all CRLF (\r\n) and CR (\r) by LF (\n)
        return re.sub(r"\r", "\n", re.sub(r"\r\n", "\n", content))
```

Reply:
Make sense. Tx for the suggestion!